src/api/comments.ts

```diff
@@ -0,0 +1,20 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: Comment['postId']) => {
+  return client.get<Comment[]>(`
+/comments?postId=${postId}`);
+};
+
+export const createComment = ({
+  postId,
+  name,
+  email,
+  body,
+}: Omit<Comment, 'id'>) => {
```

Comment:
Consider adding error handling to the `createComment` function to manage potential API request failures gracefully.